.travis.yml

```diff
@@ -1,9 +1,13 @@
 language: node_js
 
-node_js:
-  - "8.10"
-  - "lts/*"
-  - "12"
+jobs:
```

Comment:
There is already a `jobs` section few lines below

Reply:
🤦